### PR TITLE
Update grafana-monitor-best-practices.md

### PR DESCRIPTION
**Pull request by kissmydb**
### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [x] v3.1 (TiDB 3.1 versions)
- [x] v3.0 (TiDB 3.0 versions)
- [x] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

File: best-practices/grafana-monitor-best-practices.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ Prometheus 是一个拥有多维度数据模型和灵活查询语句的时序数
 
 ![TiDB 监控整体架构](/media/prometheus-in-tidb.png)
 
-从 TiDB 2.1.3 版本开始，监控采用 pull 的方式，而之前版本采用的是 push 的方式，这是一个非常好的调整，它有以下几个优点：
+从 TiDB 2.1.3 版本开始，监控可以支持 pull（默认是 push）的方式，而之前版本采用的是 push 的方式，这是一个非常好的调整，它有以下几个优点：
```
Comment (scsldb) [inline best-practices/grafana-monitor-best-practices.md:18]: 我记得只主推 push 方式，建议可以不提 pull 方式

Comment (yikeke) [reply best-practices/grafana-monitor-best-practices.md:18]: @kissmydb

Comment (kissmydb) [inline best-practices/grafana-monitor-best-practices.md:18]:
```suggestion
从 TiDB 2.1.3 版本开始，监控可以支持 pull，这是一个非常好的调整，它有以下几个优点：
```

**Pull request merged**